src/Components/Artist/MarketInsights/MarketInsights.tsx

```diff
@@ -195,14 +182,19 @@ export class MarketInsights extends React.Component<MarketInsightsProps, null> {
   }
 
   render() {
-    return (
-      <MarketInsightsContainer>
-        {this.renderAuctionHighlight()}
-        {this.renderGalleryRepresentation()}
-        {this.renderPermanentCollection()}
-        {this.hasSections() ? this.renderFeedbackLine() : null}
-      </MarketInsightsContainer>
-    )
+    if (this.hasSections()) {
+      return (
+        <MarketInsightsContainer>
+          <MarketInsightsDataContainer>
+            {this.renderAuctionHighlight()}
+            {this.renderGalleryRepresentation()}
+            {this.renderPermanentCollection()}
+          </MarketInsightsDataContainer>
+          {this.renderFeedbackLine()}
```

Comment:
Oh I see!  I wonder in that case then if it would make sense to use the `MarketDataSummary` on line 188-192 rather than repeating the logic in this component?

Reply:
The tricky part here is that the logic is slightly different and the display also has different styles. So for now I'm not combining anything here. But I think we can refactor each line into separate component as you were suggesting at some point and then assemble the display in MarketData and MarketInsights from components.